### DISCUSSION TITLE
Roadmap: Add a calender overview

### DISCUSSION THREAD

**Discussion by JordanKnott** [JordanKnott/taskcafe#15] [Ideas]
A planned feature for this project.

__Description__

A calendar tab (next to board tab) that would show a calendar displaying scheduled tasks.

Unscheduled tasks would sit in a drawer, and could be dragged & dropped onto a day in order to schedule them (at a due date).

**Comment by rilexus** [comment]
Awesome +1

**Comment by obbardc** [comment]
I like the idea of longstanding tasks which take up multiple days - so a due date and a start date would span all of the days between.

**Comment by focalfury** [comment]
I really like the idea of a global calendar where you can see at a glance your different boards tasks and when they are coming up due.  This will help and encourage people to make more boards for very specific items/projects/areas but still be able to keep track of their overall work at a larger scale!

**Comment by JordanKnott** [comment]
@obbardc One of the planned features is to allow each task to have a start and an end date. Or just a start date. Or just an end date. Or no date. 

Just not sure how the above will look in the calendar. The end date and the start & end date tasks will be easy to show, just unsure on the start date only tasks. 

@focalfury something similar to this is planned with the idea of being able to group together projects and easy see the status of them at once

**Comment by almereyda** [comment]
Start-date-only tasks could be displayed with a narrow, almost single pixel line that runs through every day.
